.coderabbit.yml

```
@@ -0,0 +1,127 @@
+    pylint:
+      enabled: true
+      args:
+        - "--max-line-length=100"
+        - "--disable=missing-docstring"  # Ruff handles docstrings; pylint focuses on other checks
+
+    ast-grep:
+      enabled: true
+      patterns: ".ast-grep.yml"  # Reference AST patterns for disallowed constructs (print, console.log, etc.)
+
+
+
```

YAMLlint (1.37.1)

[warning] too many blank lines (3 > 2)

(empty-lines)